src/components/Comments/Comment.js

```diff
@@ -359,7 +359,7 @@ const styles = StyleSheet.create({
   btnReplyText: {
     fontFamily: fonts.inter[400],
     fontSize: 13,
-    color: '#C4C4C4',
+    color: COLORS.gray1,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hard-coded color value in the `btnReplyText` style has been replaced with a dynamic color from the theme object. This improves consistency and maintainability of the code.
```diff
-    color: '#C4C4C4',
+    color: COLORS.gray1,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->